db/migrate/20160222091312_remove_other_accreditation.rb

```diff
@@ -0,0 +1,12 @@
+class RemoveOtherAccreditation < ActiveRecord::Migration
+  class Accreditation < ActiveRecord::Base
+  end
+
+  def up
+    Accreditation.where(name: 'Other').destroy_all
+  end
+
+  def down
+    fail ActiveRecord::IrreversibleMigration
```

Comment:
"If the app will break if you rollback passed this migration then the exception is the right thing, otherwise a comment is preferable (IMHO)." ;)